service-worker.tmpl

```diff
@@ -34,10 +34,8 @@
 importScripts(<%= importScripts %>);
 <% } %>
 
-/* eslint-disable quotes, comma-spacing */
 var PrecacheConfig = <%= precacheConfig %>;
-/* eslint-enable quotes, comma-spacing */
-var CacheNamePrefix = 'sw-precache-<%= version %>-<%= cacheId %>-' + (self.registration ? self.registration.scope : '') + '-';
+var CacheName = 'sw-precache-<%= version %>-<%= cacheId %>-' + (self.registration ? self.registration.scope : '');
```

Comment:
Google's style guide says that starting with a capital letter is reserved for classes (well, constructors).